src/markdown-pages/add-ons/rook.md

```diff
@@ -82,5 +82,5 @@ For example, to upgrade from `1.4.3` to `1.6.11`, you must first install `1.5.10
 If the currently installed Rook version is `1.0.x`, upgrades to both `1.4.9` and `1.5.x` are supported through the main installer.
 Alternatively, the upgrade from `1.0.x` to `1.4.9` can be triggered independently with `curl https://k8s.kurl.sh/latest/tasks.sh | sudo bash -s rook_10_to_14`.
 This upgrade migrates data off of any hostpath-based OSDs in favor of block device-based OSDs and upgrades through Rook `1.1.9`, `1.2.7` and `1.3.11` before installing `1.4.9` (and then optionally `1.5.x`).
-The upstream Rook project introduced a requirement for block storage in versions 1.3.x and later. 
-This upgrade currently only works on internet-connected installations.
+The upstream Rook project introduced a requirement for block storage in versions 1.3.x and later.
+In instances without internet access, this requires supplying an additional file when prompted.
```

Comment:
I don't want to give the actual URL because it is versioned like other kURL addons - and so an old version probably will not work with every future kurl mainscript and vise versa